ibc/types.go

```diff
@@ -22,6 +26,8 @@ type ChainConfig struct {
 	ChainID string `yaml:"chain-id"`
 	// Docker images required for running chain nodes.
 	Images []DockerImage `yaml:"images"`
+	// https://github.com/informalsystems/CometMock usage.
+	CometMockImage []DockerImage `yaml:"comet-mock-image"`
```

Comment:
Can maybe just include this in the Images?
 ( dd an optional tag to the DockerImage for 'type' so forks of CometMock can be used while we set the correct values when iterating the sidecars start)

Reply:
ictest does not like this when the standard ChainName is being used. Would be bad UX to require the image Repos and correct ordering